packages/cf-style-provider/src/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component, PropTypes, Children } from 'react';
+import { createRenderer as createFelaRenderer } from 'fela';
+import prefixer from 'fela-plugin-prefixer';
+import fallbackValue from 'fela-plugin-fallback-value';
+import unit from 'fela-plugin-unit';
+import lvha from 'fela-plugin-lvha';
+import fontRenderer from 'fela-font-renderer';
+import validator from 'fela-plugin-validator';
+import beautifier from 'fela-beautifier';
+import monolithic from 'fela-monolithic';
+import { Provider } from 'react-fela';
+import { variables } from 'cf-style-const';
+import { ThemeProvider } from 'cf-style-container';
+
+export const createRenderer = opts => {
+  const defaultOpts = {
+    selectorPrefix: 'cf-',
+    dev: false
+  };
+  const usedOpts = Object.assign({}, defaultOpts, opts);
+  const plugins = [prefixer(), fallbackValue(), unit(), lvha()];
+  const enhancers = [fontRenderer(usedOpts.fontNode)];
+
+  if (usedOpts.dev === true) {
+    plugins.push(validator());
+    enhancers.push(beautifier());
+    enhancers.push(monolithic());
+  }
+
+  return createFelaRenderer({
+    plugins,
+    enhancers,
+    selectorPrefix: [usedOpts.selectorPrefix]
+  });
+};
+
+export const createProvider = opts => {
```

Comment:
Because you need to configure the [renderer](https://github.com/rofrischmann/fela/blob/master/docs/basics/Renderer.md). It doesn't seem like something that should be done in `render()` ?